api/src/legacy_soap_api/legacy_soap_api_client.py

```diff
@@ -150,21 +150,16 @@ def log_diffs(self, proxy_response_soap_dict: dict, simpler_response_soap_dict:
                 },
             )
 
-    def get_simpler_soap_response(self, proxy_response: SOAPResponse) -> tuple:
+    def get_simpler_soap_response(
+        self, proxy_response: SOAPResponse, compare: bool = False
+    ) -> tuple:
```

Comment:
Where does compare get set/configured? We'd want a configuration per operation saying whether to do a comparison or not.